core/src/main/com/github/mmorihiro/core/ArrayActor.kt

```diff
@@ -0,0 +1,33 @@
+package com.github.mmorihiro.core
+
+import com.badlogic.gdx.graphics.g2d.Batch
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer
+import com.badlogic.gdx.scenes.scene2d.Actor
+
+
+class ArrayActor : Actor() {
```

Comment:
I forgot adding `internal` modifier.
I'll change the settings of intellij's template for class.